crates/payload/basic/src/lib.rs

```diff
@@ -648,7 +648,7 @@ pub struct PayloadConfig {
     /// The chain spec.
     chain_spec: Arc<ChainSpec>,
     /// The rollup's compute pending block configuration option.
-    /// TODO(clabby): Implement this feature.
+    // TODO(clabby): Implement this feature.
```

Comment:
this will cause a conflict with #5876 

```suggestion
    /// TODO(clabby): Implement this feature.
```